modules/heat_conduction/src/materials/ElectricalConductivity.C

```diff
@@ -16,32 +16,35 @@ InputParameters
 ElectricalConductivity::validParams()
 {
   InputParameters params = Material::validParams();
-  params.addCoupledVar("temp", 300.0, "variable for temperature");
+  params.addCoupledVar("temperature", 293.0, "Coupled variable for temperature");
   params.addParam<std::string>("base_name", "Material property base name");
   params.addParam<Real>("length_scale", 1.0e-9, "Length scale of model");
   params.addParam<Real>(
-      "ref_resistivity",
-      6.5e-6,
+      "reference_resistivity",
+      5.6e-8,
       "Electrical resistivity of the material at reference temperature in ohm-m.");
   params.addParam<Real>(
```

Comment:
I don't remember if the default values were for any specific material. Feel free to change it. We probably could make copper the default material. Also, it probably would be good to put in a comment on what material the default values relate to.

Reply:
I do have comments in the class description, as well as the documentation. I was trying not to modify the source code *too* much, since that wasn't the point of the PR, but it was a bit unavoidable if I wanted to make it more clear. Copper generally better fits the linear approximation for a wider range of temperature variation, so I may just go with that instead of tungsten (which if I remember right is more of a power law).

Reply:
Updated defaults to copper and modified the input file accordingly - also added a few more clarifying comments.